### DISCUSSION TITLE
The category "uncategorized" is weird

### DISCUSSION THREAD

**Discussion by tehabe** [FreshRSS/FreshRSS#5414] [Q&A] [answered]
I tried FreshRSS today and it is good, but one thing is odd to me, the category "uncategorized" doesn't make sense to me (anymore). I mean if the feed is not part of any category it shouldn't be part of any category, not even "uncategorized".

Is there a way to achieve this?

**Comment by aledeg** [comment, marked as answer]
Actually, it's not possible. The reason behind this is that the category is a mandatory element of each feed. Thus the use of the "uncategorized" category which is a misnomer. It should be "default category" or something that conveys exactly what it does.

That category is the one created automatically by the system to ensure you can add feeds. It also simplifies the UI since we do not have to deal with loose feeds. It makes the whole thing more consistent.
